### DISCUSSION TITLE
Not receiving student rights

### DISCUSSION THREAD

**Discussion by hsyfly** [community/community#141454] [GitHub Education]
### Hi everyone,

I have been certified as a student for several days now, but I have not yet received my student rights. I would like to know approximately when I will receive my rights. thank you!
<img width="1159" alt="GitHub" src="https://github.com/user-attachments/assets/feb1acc0-770b-4c74-b9fc-a37d754e491d">

**Comment by davevad93** [comment]
Hi @hsyfly ,

Given your status, you should already have access to your benefits. Try first going to the [pack offers page](https://education.github.com/pack), scroll down to the "***All offers***" section and select one of the available offers, for example [Copilot](https://education.github.com/pack/redeem/copilot-student), if you can redeem it you are fine, if you can't, I suggest you to raise a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue.

**Reply by hsyfly** [reply]
Okay, thank you! I still haven't received my benefits; I have already submitted the issue!
